lib/tasks/import.rake

```diff
@@ -7,38 +7,6 @@ namespace :import do
     end
   end
 
-  namespace :hets do
-    desc 'Import the hets library.'
-    task :lib => :environment do
-      def find_hets_lib_path
-        settings_for_development['paths']['hets']['hets_lib'].
-          map { |path| File.expand_path path }.
-          find { |path| File.directory?(path) }
-      end
-
-      hets_lib_path = ENV['HETS_LIB']
-      hets_lib_path ||= find_hets_lib_path
-      unless File.directory?(hets_lib_path)
-        raise 'No path to hets-lib given or it is not a directory. '\
-              'Please specify the path to hets-lib in the environment '\
-              'variable HETS_LIB.'
-      end
-
-      user   = User.find_by_email! ENV['EMAIL'] unless ENV['EMAIL'].nil?
-      user ||= User.find_all_by_admin(true).first
-
-      repo = Repository.new(name: 'Hets lib')
-
-      begin
-        repo.save!
-      rescue ActiveRecord::RecordInvalid
-        abort '"Hets lib" repository already existing.'
-      end
-
-      repo.import_ontologies(user, hets_lib_path)
```

Comment:
This is the only usage of the method `import_ontologies`. Please delete it (and `import_ontology`) in `app/models/repository/ontologies.rb` as well.